modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -9034,6 +9218,65 @@ public void testPostRelatedObjectEntryInDifferentCompany()
 		);
 	}
 
+	@Test
+	public void testPostScopeScopeKey() throws Exception {
+		Group group = _groupLocalService.fetchGroup(
+			TestPropsValues.getGroupId());
+
+		// Using groupExternalReferenceCode
+
+		int objectFieldValue = RandomTestUtil.randomInt();
+
+		Assert.assertEquals(
+			objectFieldValue,
+			HTTPTestUtil.invokeToJSONObject(
+				JSONUtil.put(
+					_OBJECT_FIELD_NAME_1, objectFieldValue
+				).toString(),
+				_getEndpoint(
+					group.getExternalReferenceCode(), 0,
+					_siteScopedObjectDefinition1),
+				Http.Method.POST
+			).getInt(
+				_OBJECT_FIELD_NAME_1
+			));
```

Comment:
We should avoid chaining method calls. Have you seen any examples of this pattern in our existing tests?

Reply:
We have this here: https://github.com/liferay/liferay-portal/blob/6fe4ba20a08a2d9003f0809656d512b4990ce309/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java#L9488C3-L9496C16.


If we decide to not chain the method calls we can't "inline" calls and the asserts, right?